CONTRIBUTORS.MD

```diff
@@ -105,7 +105,7 @@ operates according to the rules specified in the
 | Thomas Morrell | Caltech Library | [@tmorell](https://github.com/tmorrell) |
 | Valentin Lorentz | Software Heritage | [@progval](https://github.com/progval) |
 
-## V3.1 contributors
+## V3.1 contributors (Sep 30, 2025)
```

Comment:
I don't think we know the release date yet.

Reply:
I put the date of the latest PR merge